### ISSUE TITLE
Issues with Android Studio 3.3 beta

### ISSUE THREAD

**Issue by eygraber** [pbreault/adb-idea#98]
1. The shortcut (ctl+alt+shft+a) was overwritten to the refactor action "Introduce Type Alias"

2. Manually opening the dialog and selecting any option results in a message being printed:

    `ADB IDEA: Couldn't determine if a gradle sync is in progress`

**Comment by chainchelliah** [comment]
Yeah.. 3.3 stable version also not working. Any update on this team. @pbreault

**Comment by pbreault** [comment]
Looks like the Kotlin plugin overrides Adb Idea's default shortcut on Windows/Linux.
I'll see what kind of shortcut would make more sense on those platforms (suggestions welcomed!).

Meanwhile, you can remap the shortcut manually in your keymap.

Do you still see an error message when invoking the commands directly? If so, what platform are you on and does it fail on every project?

**Comment by shahsurajk** [comment]
For Android Studio 3.3 and above a new shortcut is added for the keymap `ctrl+alt+shift+a` here are possible workarounds for using this plugin: 

1. Shift the focus from the main editor, open logcat or run window or move the focus of your cursor to project directory, then try the same keymap binding
2. Go the settings/ preferences `ctrl+alt+s` for linux, go to keymap, Main Menu -> Refactor -> Extract -> Type Alias.. (Right click on this and remove this shortcut)
